components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -110,6 +111,11 @@
                   <a [routerLink]="['/infrastructure','chef-servers', server?.id, 'organizations', org.id]">{{ org.name }}</a>
                 </chef-td>
                 <chef-td>{{ org.admin_user }}</chef-td>
+                <chef-td class="projects-column">
+                  <ng-container *ngIf="org.projects.length === 0">{{ unassigned }}</ng-container>
+                  <ng-container *ngIf="org.projects.length === 1">{{ org.projects[0] }}</ng-container>
+                  <ng-container *ngIf="org.projects.length > 1">{{ org.projects.length }} projects</ng-container>
+                </chef-td>
```

Comment:
I recently learned of a cleaner way to handle this in the template using `ngSwitch`. 

https://angular.io/api/common/NgSwitch

You can see it in action here: https://github.com/chef/automate/pull/3670/files#diff-0375bab3040117dbb76df46a8d950b4bR17-R22